agenta-cli/pyproject.toml

```diff
@@ -19,15 +19,15 @@ keywords = ["LLMOps", "LLM", "evaluation", "prompt engineering"]
 python = "^3.9"
 docker = ">=6.1.1,<8.0.0"
 click = "^8.1.3"
-fastapi = ">=0.96.1"
+fastapi = "^0.111.0"
```

Comment:
I would not pin this, since it will result in lots of issues for our users if their projects use another version, just let's make it >= 100 (and test it with that version)

Reply:
Got it.